src/docfx/restore/PackageFetchOptions.cs

```diff
@@ -22,5 +22,10 @@ internal enum PackageFetchOptions
         /// Ignore the package resolved directory not existed error
         /// </summary>
         IgnoreDirectoryNonExisted = 0b0010,
+
+        /// <summary>
+        /// Ignore the branch fallback when restore the package
+        /// </summary>
+        IgnoreBranchFallback = 0b0100,
```

Comment:
`IgnoreBranchFallback` sounds like not doing branch fallback at all.
```suggestion
        IgnoreBranchFallbackError = 0b0100,
```